fastapi/dependencies/utils.py

```diff
@@ -84,7 +85,22 @@
 )
 
 
+class CallType(IntEnum):
+    ASYNC = 0
+    GENERATOR = 1
+    THREADPOOL = 2
+    SYNC = 3
+
+
```

Comment:
I suggest assigning string values rather than integers to evade conflicts.

Reply:
This is internal enum, no one will use it. IntEnum might be slightly faster.